src/content/en/updates/2017/12/audio-worklet.md

```diff
@@ -2,45 +2,42 @@ project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
 
 {# wf_published_on: 2017-12-14 #}
-{# wf_updated_on: 2017-12-14 #}
+{# wf_updated_on: 2018-03-15 #}
 {# wf_featured_image: /web/updates/images/generic/audio.png #}
-{# wf_tags: chrome64,webaudio #}
+{# wf_tags: chrome64,chrome66,webaudio #}
 {# wf_featured_snippet: Chrome 64 comes with a highly anticipated new feature in Web Audio API - AudioWorklet. AudioWorklet nicely keeps the user-supplied JavaScript code all within the audio processing thread — that is, it doesn’t have to jump over to the main thread to process audio. #}
 {# wf_blink_components: Blink>WebAudio #}
 
 # Enter AudioWorklet {: .page-title }
 
 {% include "web/_shared/contributors/hongchanchoi.html" %}
 
+<aside class="note">
+  <p><b>Note:</b> AudioWorklet is enabled by default in Chrome 66.</p>
```

Comment:
This can just be:

```
Note: AudioWorklet is enabled by default in Chrome 66.
```

All the styling will be done for you :)

Reply:
Thanks! Done.